docs/devguide/fields-yml.asciidoc

```diff
@@ -29,27 +30,35 @@ Define the field mappings in the `fields` array:
 ----------------------------------------------------------------------
 
 <1> `name`: The field name
-<2> `type`: The field type. The value for the `type` key can be any of the datatypes https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-types.html[available in Elasticsearch]. If no value is specified, the field will default to being a `keyword`.
+<2> `type`: The field type. The value of `type` can be any datatype {ref}/mapping-types.html[available in {es}]. If no value is specified, the default type is `keyword`.
 <3> `required`: Whether or not a field value is required
 <4> `description`: Some information about the field contents
 
 ==== Mapping parameters
-Other mapping parameters can be specified for each field. Consult the https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-params.html[Elasticsearch reference] for more details on each of these parameters.
+
+You can specify other mapping parameters for each field. See the
+{ref}/mapping-params.html[{es} Reference] for more details about each
+parameter.
 
 [horizontal]
 `format`:: Specify a custom date format used by the field.
-`multi_fields`:: For `text` or `keyword` fields, `multi_fields` can be used to define multi-field mappings.
+`multi_fields`:: For `text` or `keyword` fields, use `multi_fields` to define
+multi-field mappings.
 `enabled`:: Whether or not the field is enabled.
 `analyzer`:: Which analyzer to use when indexing.
 `search_analyzer`:: Which analyzer to use when searching.
 `norms`:: Applies to `text` and `keyword` fields. Default is `false`.
-`dynamic`:: Dynamic field control. Can be one of `true` (default), `false` or `strict`.
+`dynamic`:: Dynamic field control. Can be one of `true` (default), `false`, or
+`strict`.
 `index`:: Whether or not the field should be indexed.
-`doc_values`:: Whether or not the field should have doc values generated. See docs.
+`doc_values`:: Whether or not the field should have doc values generated.
```

Comment:
Perhaps bring back the "see docs" (in spirit anyway) and link to https://www.elastic.co/guide/en/elasticsearch/reference/current/doc-values.html?

Reply:
On further thought, I suppose you could link off to ES docs for each of these mapping parameters. So I suppose you'd either way to do it for all of them or none of them, not just this one.

Reply:
There's already a link to the docs on line 40. I think the original author just forgot to remove the text.